### PR TITLE
Sign up multiple events help text

### PR DESCRIPTION
**Pull request by caleybrock**
screen shot 2015-10-29 at 6 16 14 pm

### CODE REVIEW

**Comment by tanyaparker** [comment]
Could you apply css to the asterisk to make it red? like on https://code.org/volunteer. Or would that stand out too much?

**Comment by tanyaparker** [comment]
Actually, I think it might be fine to have the asterisk separate from the error message. Append it to the fields and prepend it to the error message.

**Comment by caleybrock** [comment]
I think that might make it confusing what to look at when the user actually has an error.

**Comment by caleybrock** [comment]
Maybe I could use another colour if you think it needs to stand out more? I just think that red makes it seem like the user is doing something wrong. Especially since even our error messages aren't red.

File: pegasus/sites.v3/hourofcode.com/public/css/front-page.css [hunk 1]
```diff
@@ -114,3 +114,12 @@
...
 }
+
+.multiple-event-warning {
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.ast {
+  color: #008b9a;
+}
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.css]: these pesky new lines!

File: pegasus/sites.v3/hourofcode.com/public/css/front-page.css [hunk 2]
```diff
@@ -114,3 +114,12 @@
 .microsoft {
   width: 160px;
 }
+
+.multiple-event-warning {
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.ast {
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.css]: I would rename this to something like `required-asterisk`. Just reading this css out of context it's unclear what ast is or why it's colored.

**Comment by tanyaparker** [comment]
Just those nits then looks good!

**Comment by caleybrock** [comment]
cool, thanks :)

**Pull request merged**